### PR TITLE
Refactors /say code to /tg/ standard

### PR DESCRIPTION
**Pull request by DominikPanic**
Only one 1 hack! Silicon code is on the list next.

### CODE REVIEW

File: code/game/say.dm [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+/proc/message_spans_start(list/spans)
+	var/output = "<span class='"
+	for(var/S in spans)
+		output = "[output][S] "
+	output = "[output]'>"
+	return output
+
+
+/proc/say_test(text)
```
Comment (Menshin) [inline code/game/say.dm]: ?

Comment (DominikPanic) [reply code/game/say.dm]: `var/image/I = image('icons/mob/talk.dmi', src, "[bubble_type][say_test(message)]", FLY_LAYER)`

File: code/game/objects/items/radio/radio.dm [hunk 2]
```diff
@@ -403,12 +405,26 @@
...
-	if (broadcasting)
-		if(get_dist(src, M) <= canhear_range)
-			talk_into(M, msg, null, verb, language)
+/obj/item/radio/Hear(message, atom/movable/speaker, message_language, raw_message, radio_freq, list/spans, message_mode)
+	. = ..()
+
+	if(radio_freq || !broadcasting || get_dist(src, speaker) > canhear_range)
+		return
+
+	if(message_mode == MODE_WHISPER || message_mode == MODE_WHISPER_CRIT)
```
Comment (Rohesie) [inline code/game/objects/items/radio/radio.dm]: Could use a simple `switch(message_mode)`, but don't bother if you think it's too much effort.

Comment (DominikPanic) [reply code/game/objects/items/radio/radio.dm]: Yeah this will get replaced in the next PR anyway

File: code/game/say.dm [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+	"[MED_FREQ]" = "medradio",
+	"[SUP_FREQ]" = "supradio"
+	))
+
+
+/atom/movable/proc/say(message, bubble_type, list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null)
+	if(!can_speak())
+		return
+
+	if(message == "" || !message)
```
Comment (Rohesie) [inline code/game/say.dm]:
```suggestion
	if(!message)
```
Potato, potato.

Comment (DominikPanic) [reply code/game/say.dm]: Ironically enough /tg/ code xP Committing

File: code/modules/mob/living/silicon/decoy/decoy.dm [hunk 4]
```diff
@@ -65,5 +77,5 @@
 		if("broadcast")
 			message_mode = MODE_HEADSET
 
-	ai_headset.talk_into(src, message, message_mode)
+	ai_headset.talk_into(src, message, message_mode, , language)
```
Comment (Menshin) [inline code/modules/mob/living/silicon/decoy/decoy.dm]: Don't leave a blank, it's error prone.
Better to use a named argument.

**Pull request merged**